Adds documentation for checklist to do for production ready installation

andreslucena opened the pull request:
#### :tophat: What? Why?

There are some newcommers to the project that don't understand that well all that you need to know to have a production ready installation. This tries to solve this problem with a checklist of things you need to have before going live. 

#### :pushpin: Related Issues
*None*

#### :clipboard: Subtasks
*None*

### :camera: Screenshots (optional)
*None*

oriolgual commented on docs/checklist.md:
```diff
@@ -0,0 +1,42 @@
+
+1. Choose a **domain** for your application. Some typical names involve "Participation" or "Decision" conjugations.
+
+1. Choose which **languages** do you want for your application. In case that your language isn't supported you should translate it on [Crowdin](https://crowdin.com/).
```
Maybe link to our Crowdin project?

oriolgual commented on docs/checklist.md:
```diff
@@ -0,0 +1,42 @@
+
+1. Customize the **look and feel** (colors, pictures, fonts, etc).
+
+1. Configure **SSL**. We recommend using at least [Let's Encrypt](https://letsencrypt.org/) for a minimun security.
```
s/minimun/minimum

oriolgual commented on docs/checklist.md:
```diff
@@ -0,0 +1,42 @@
+
+1. Comply with your current **legal requirements**, like to registrate your privacy policy with the autorities (eg LOPD on Spain).
+
+1. Fill the **Participatory Processes Configuration Form** for preparing your Participatory Process for Decidim.
```
s/for preparing/to prepare

andreslucena commented:
Ok, I've just applied all the changes proposed by @oriolgual

andreslucena commented:
Also, it'd be nice to review this checklist by everyone  cc/ @decidim/developers @decidim/product @decidim/docs

agustibr commented on docs/checklist.md:
```diff
@@ -0,0 +1,42 @@
+
+1. Choose which **languages** do you want for your application. In case that your language isn't supported you should translate it on [Crowdin](https://crowdin.com/project/decidim).
+
+1. Customize the **look and feel** (colors, pictures, fonts, etc).
```
maybe I would add a link to [docs/customization/styles.md](https://github.com/decidim/decidim/blob/master/docs/customization/styles.md)

andreslucena replied:
:+1:

mrcasals commented:
@andreslucena I've solved some conflicts, please check everything is OK

deivid-rodriguez commented:
ping @andreslucena Is this abandoned?

tramuntanal commented on docs/checklist.md:
```diff
@@ -0,0 +1,42 @@
+
+1. Choose which **languages** do you want for your application. In case that your language isn't supported you should translate it on [Crowdin](https://crowdin.com/project/decidim).
+
+1. Customize the **look and feel** (colors, pictures, fonts, etc).
```
Hey, It would be great to have an exact list of what can be customized and how. For example:
- primary-color (in variables.scss): action buttons color, omnipresent banner, links, icons
- secondary-color (in variables.scss): .....
- it would also be great to have background-header-color background-body-color variables.
- etc..

andreslucena replied:
ATM we're gonna link to https://github.com/decidim/decidim/blob/master/docs/customization/styles.md as @agustibr said, if we want to improve the styling/css docs we can do that there.

tramuntanal commented:
Hi,
We find also interesting
- to add a check for delayed_job installation (or alternative backends).
- filling options in decidim initializer
- install postgres
- install trigram extension in postgres for decidim's database (i think that now it is a default requirement)

andreslucena commented:
@deivid-rodriguez sorry I thought it was already merged. 

I've added some more checks and corrections with feedback from @tramuntanal and @agustibr - thanks!

Added: 
* Link to customizing styles 
* Add decidim initializer
* Add ActiveJob link
* Add social providers integration link

oriolgual commented:
Is this ready to be merged then @andreslucena ?

andreslucena commented:
@oriolgual yes, it's ready to be merged
